### PR TITLE
Pete/flip bitorder

### PR DESCRIPTION
**Pull request by pdietl**

### CODE REVIEW

File: firmware/modes/bitbang/main.c [hunk 1]
```diff
@@ -15,29 +15,46 @@ static inline void print_help(void)
...
+    com_println("           LSB is ZIF pin 1");
     //com_println("P val      VPP: get active pins (VPP_READ)");
     com_println("VDD");
-    com_println("e val      VDD: enable and/or disable (VDD_DISABLE/VDD_ENABLE)");
-    com_println("v val      VDD: set voltage enum (VDD_SET)");
-    com_println("d val      VDD: set active pins (VDD_WRITE)");
+    com_println("e val      VDD: enable or disable");
+    com_println("           1 = enable, 0 = disable");
+    com_println("v val      VDD: set voltage enum");
+    com_println("           val in range [0,8]");
```
Comment (JohnDMcMaster) [inline firmware/modes/bitbang/main.c:30]: 0 to 7?

Comment (pdietl) [reply firmware/modes/bitbang/main.c:30]: oops yeah.

File: firmware/modes/bitbang/main.c [hunk 2]
```diff
@@ -15,29 +15,46 @@ static inline void print_help(void)
...
-    com_println("T          I/O: get ZIF tristate setting (ZIF_DIR_READ)");
+    com_println("t val      I/O: set ZIF tristate setting");
+    com_println("           1 = pin is driven, 0 = High Z");
+    com_println("T          I/O: get ZIF tristate setting");
+    com_println("           1 = pin is driven, 0 = High Z");
     com_println("z val      I/O: set ZIF pins (ZIF_WRITE)");
+    com_println("           val must be 10 hex digits");
+    com_println("           LSB is ZIF pin 1");
     com_println("Z          I/O: get ZIF pins (ZIF_READ)");
+    com_println("           LSB is ZIF pin 1");
```
Comment (JohnDMcMaster) [inline firmware/modes/bitbang/main.c:50]: maybe we can put some ZIF examples below, but not something to block this PR on

Comment (pdietl) [reply firmware/modes/bitbang/main.c:50]: You mean example usage?

**Comment by JohnDMcMaster** [comment]
Request small change in incorrect help message, otherwise LGTM. Thanks!

**Comment by JohnDMcMaster** [comment]
Actually: can you also do a sweep on the python code? I don't see any changes there and I imagine this will break a lot

**Comment by pdietl** [comment]
Why do you think it will break a lot?

**Comment by JohnDMcMaster** [comment]
The bit ordering was changed. Therefore the API will be broken. Take a look at this for example

https://github.com/JohnDMcMaster/open-tl866/blob/master/py/util/read_rom.py

In particular I think this is the line that must be changed to match the C change:

https://github.com/JohnDMcMaster/open-tl866/blob/master/py/otl866/aclient.py#L246

**Comment by pdietl** [comment]
I believe I addressed all the issues @JohnDMcMaster

**Comment by JohnDMcMaster** [comment]
LGTM! This is a breaking API change so I'll also follow up with a note to the readme

**Pull request merged**